.gitignore

```diff
@@ -27,6 +27,7 @@ share/python-wheels/
 MANIFEST
 .pip_cache/
 package/
+lambdas/mtls_env_certs/
```

Comment:
added here https://github.com/NHSDigital/national-document-repository/pull/872/files
Which you approved! 😆 
It's ok. Whichever goes in first.